src/main/kotlin/com/openlattice/organizations/ExternalDatabaseManagementService.kt

```diff
@@ -84,13 +87,13 @@ class ExternalDatabaseManagementService(
                 "OrganizationExternalDatabaseColumn $columnFQN already exists")
         aclKeyReservations.reserveIdAndValidateType(column, columnFQN::getFullQualifiedNameAsString)
 
-        val columnAclKey = AclKey(orgId, column.tableId, column.id)
+        val columnAclKey = AclKey(column.tableId, column.id)
         authorizationManager.setSecurableObjectType(columnAclKey, SecurableObjectType.OrganizationExternalDatabaseColumn)
 
         return column.id
     }
 
-    fun createNewColumnObjects(dbName: String, tableName: String, tableId: UUID, orgId: UUID, columnName: Optional<String>): BasePostgresIterable<OrganizationExternalDatabaseColumn> {
+    fun getColumnMetadata(dbName: String, tableName: String, tableId: UUID, orgId: UUID, columnName: Optional<String>): BasePostgresIterable<OrganizationExternalDatabaseColumn> {
```

Comment:
Since this is only used in indexer, I think it should be moved there unless we want to use it elsewhere too.